scripts/index.js

```diff
@@ -0,0 +1,117 @@
+const state = {
+    temp: 82,
+    sky: "Sunny",
+    city: "Emerald City"
+};
+
+const skys = [`☁️ ☁️ ☁️ ☀️ ☁️ ☁️`,
+    `☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️`,
+    `🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧`,
+    `🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨`
+]
+
+const landscapes = [
+    `🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂`,
+    `🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷`,
+    `🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃`,
+    `🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲`
+]
+
+const incrementTemp = () => {
+    let temp = document.getElementById("temp");
+    state.temp += 1;
+    temp.innerHTML = state.temp;
+    const landscapeIdx = getLandscapeIdx()
+
+    const landscapeImg = document.getElementById("landscapeImg");
+    landscapeImg.innerHTML = landscapes[landscapeIdx]
+}
+
+const decrementTemp = () => {
+    const temp = document.getElementById("temp");
+    state.temp -= 1;
+    temp.innerHTML = state.temp;
+
+    const landscapeIdx = getLandscapeIdx()
+    const landscapeImg = document.getElementById("landscapeImg");
+    landscapeImg.innerHTML = landscapes[landscapeIdx]
+}
+
+const getLandscapeIdx = () => {
+    if (state.temp >= 80) {
+        return 0
+    } else if (state.temp >= 70) {
+        return 1
+    } else if (state.temp >= 60) {
+        return 2
+    } else {
+        return 3
+    }
+}
+
+const getSkyIdx = () => {
+    switch (state.sky) {
+        case 'Sunny':
+            return 0
+        case 'Cloudy':
+            return 1
+        case 'Rainy':
+            return 2
+        case 'Snowy':
+            return 3
+        default:
+            break
+    }
+}
+
+function setCity() {
+    const newCity = document.getElementById("cityInput").value;
+    state.city = newCity;
+    document.getElementById("city").innerHTML = `for ✨${newCity}✨`;
+}
+
+function resetCity() {
+    state.city = "Emerald City";
+    document.getElementById("city").innerHTML = `for ✨${state.city}✨`;
```

Comment:
In the HTML markup, we can tighten our span so that the "for" and sparkles are part of the html, which would allow us to avoid needing to do string interpolation in a few places here. Unfortunately, we can't simply make a const formatting string, so if we wanted to keep this in code, a helper function might be a good option so that the formatting string could exist just once in the helper function.